.github/workflows/haskell.yml

```
@@ -66,9 +66,11 @@ jobs:
 
       - name: Compile example Agda code to Scala 2
         run: cabal run -- agda2scala --scala-dialect=Scala2 --out-dir=scala2/src/main/scala ./examples/adts.agda
+        run: cabal run -- agda2scala --scala-dialect=Scala2 --out-dir=scala2/src/main/scala ./examples/rbt.agda
```

actionlint (1.7.12)

[error] key "run" is duplicated in element of "steps" section. previously defined at line:68,col:9

(syntax-check)